src/pretix/control/templates/pretixcontrol/event/index.html

```diff
@@ -16,6 +16,42 @@ <h1>
             {% endif %}
         </small>
     </h1>
+    <div class="helper-space-below">
+        {% trans "Shop URL:" %}
+        <span id="shop_url" class="text-muted">{% abseventurl request.event "presale:event.index" %}</span>
+        <button type="button" class="btn btn-default btn-xs btn-clipboard" data-clipboard-target="#shop_url">
```

Comment:
As this is dependent on JS, add `js-only`?
```suggestion
        <button type="button" class="btn btn-default btn-xs btn-clipboard js-only" data-clipboard-target="#shop_url">
```